indexer/map_object.cpp

```diff
@@ -94,6 +94,32 @@ string MapObject::GetLocalizedType() const
   return platform::GetLocalizedTypeName(classif().GetReadableObjectName(copy.GetBestType()));
 }
 
+std::string MapObject::GetAllLocalizedTypes() const
+{
+  ASSERT(!m_types.Empty(), ());
+  feature::TypesHolder copy(m_types);
+  copy.SortBySpec();
+  
+  auto const & isPoi = ftypes::IsPoiChecker::Instance();
```

Comment:
Do multiple area features also work well? I wonder how they pass IsPoiChecker filter, or is it a not only POI filter?

Reply:
Yes, they work fine 

https://www.openstreetmap.org/way/1090710626#map=18/41.47902/2.09795

<img src="https://github.com/organicmaps/organicmaps/assets/47610359/18f5997b-596f-4689-885e-fd04c025ec6c" width="320"/>